tests/unit/test_variants.py

```diff
@@ -70,3 +73,115 @@ def testPhasesetZero(self):
 
     def testGenotypeHaploid(self):
         self.verifyGenotypeConversion("1", "376", [1], None)
+
+
+class TestAbstractVariantSet(unittest.TestCase):
+    """
+    Unit tests for the abstract variant set.
+    """
+    def setUp(self):
+        self._vsname = "testVariantSet"
+        self._backend = backend.AbstractBackend()
+        self._dataset = datasets.AbstractDataset(self._backend)
+        self._variantSet = variants.AbstractVariantSet(
+            self._dataset, self._vsname)
+
+        # create some simulated variants with dummy data
+        self._simulatedVariantSet = variants.SimulatedVariantSet(
+            self._dataset, self._vsname, randomSeed=0,
+            numCalls=1, variantDensity=1)
+        self._simulatedVariants = self._simulatedVariantSet.getVariants(
+            'ref', 100, 110, ["nope"])
+        self._real_variant = list(self._simulatedVariants)[0]
+
+    def testAddOneCallSet(self):
+        self.assertEqual(self._variantSet.getNumCallSets(), 0)
+        callSetName = "callSetName"
+        self._variantSet.addCallSet(callSetName)
+        self.assertEqual(self._variantSet.getNumCallSets(), 1)
+        callSet = self._variantSet.getCallSetByIndex(0)
+        self.assertEqual(
+            self._variantSet.getCallSetByName(callSetName), callSet)
+        self.assertEqual(
+            self._variantSet.getCallSet(callSet.getId()), callSet)
+        self.assertEqual(self._variantSet.getCallSets(), [callSet])
+
+    def testAddMultipleCallSets(self):
+        # disable maxdiff value in assertEqual because list
+        # of 10 variants was too large
+        self.assertEqual.__self__.maxDiff = None
+        self.assertEqual(self._variantSet.getNumCallSets(), 0)
+        callSetName = "callSetName{}"
+        callSetCount = 10
+        allCallSets = []
+        for x in range(0, callSetCount):
```

Comment:
use `i` as the counting variable

use `range(callSetCount)` -- the 0 argument is implied